app/src/main/java/org/dicekeys/app/data/BackupRecipes.kt

```diff
@@ -1,10 +1,34 @@
 package org.dicekeys.app.data
 
+import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 import org.dicekeys.api.DerivationRecipe
+import org.dicekeys.crypto.seeded.DerivationOptions
 
 @Serializable
-data class BackupRecipes(
-    val version: Int,
-    val recipes: List<DerivationRecipe>
-)
\ No newline at end of file
+data class BackupRecipe(
+    @SerialName("name")
+    val name: String,
+
+    @SerialName("type")
+    val type: DerivationOptions.Type,
+
+    @SerialName("recipeJson")
+    val recipeJson: String
+) {
+    companion object {
+
+        fun fromDerivationRecipes(recipes: List<DerivationRecipe>): List<BackupRecipe> {
+            return recipes.map {
+                BackupRecipe(name = it.name, type = it.type, recipeJson = it.recipeJson)
+            }
+
+        }
+    }
+}
+
+fun List<BackupRecipe>.toDerivationRecipes(): List<DerivationRecipe> {
+    return map {
+        DerivationRecipe(name = it.name, type = it.type, recipeJson = it.recipeJson)
```

Comment:
If they share the exact same fields, why have two different classes for them?

Reply:
The `DerivationRecipe` class uses the key "derivation_options_json" to serialize it. That's the only difference i think.

```
constructor(
        @SerialName("type")
        val type: DerivationOptions.Type,
        @SerialName("name")
        val name: String,
        @SerialName("derivation_options_json")
        val recipeJson: String
)
```

Reply:
If the JSON engine allows you to use an alternate name for reading, we should transition to the new name.

Reply:
https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/json.md

Reply:
I believe `recipeJson`.  Why would you think it would be `recipe_json`?

Reply:
1. I can use `@JsonNames("derivation_options_json")` to keep backwards compatibility, thanks for sharing.
2. Just to be sure. We have mixed camelcase and underscores so wanted to be sure.